WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -97,37 +116,53 @@ private extension ProductInventorySettingsViewController {
 //
 private extension ProductInventorySettingsViewController {
     func reloadSections() {
-        let stockSection: Section
-        if manageStockEnabled {
-            stockSection = Section(rows: [.manageStock, .stockQuantity, .backorders])
-        } else {
-            stockSection = Section(rows: [.manageStock, .stockStatus])
-        }
+        let sections: [Section]
+        switch formType {
+        case .inventory:
+            let stockSection: Section
+            if manageStockEnabled {
+                stockSection = Section(rows: [.manageStock, .stockQuantity, .backorders])
+            } else {
+                stockSection = Section(rows: [.manageStock, .stockStatus])
+            }
 
-        let skuSection: Section
-        if let error = error {
-            skuSection = Section(errorTitle: error.alertMessage,
-                                 rows: [.sku])
-        } else {
-            skuSection = Section(rows: [.sku])
+            sections = [
+                createSKUSection(),
+                stockSection,
+                Section(rows: [.limitOnePerOrder])
+            ]
+        case .sku:
+            sections = [
+                createSKUSection()
+            ]
         }
 
-        sections = [
-            skuSection,
-            stockSection,
-            Section(rows: [.limitOnePerOrder])
-        ]
+        self.sections = sections
 
         tableView.reloadData()
     }
+
+    func createSKUSection() -> Section {
+        if let error = error {
+            return Section(errorTitle: error.alertMessage,
+                           rows: [.sku])
```

Comment:
I would write everything on the same line of code in this case.

Reply:
yup! updated in: https://github.com/woocommerce/woocommerce-ios/pull/2437/commits/c2f702290f49a21411d46a5074200ff271a74bab